test/test-fs.c

```diff
@@ -115,6 +115,15 @@ static char test_buf[] = "test-buffer\n";
 static char test_buf2[] = "second-buffer\n";
 static uv_buf_t iov;
 
+#ifdef _WIN32
+/*
+ * This tag and guid have no special meaning, and don't conflict with
+ * reserved ids.
+*/
+static unsigned REPARSE_TAG = 0x9913;
+static GUID REPARSE_GUID = { 0x1bf6205f, 0x46ae, 0x4527, 0xb1, 0x0c, 0xc5, 0x09, 0xb7, 0x55, 0x22, 0x80 };
```

Comment:
Long line.  Can you wrap this at 80 columns?

Reply:
Sure, no problem.